editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -867,4 +891,28 @@ pub mod tests {
 
         assert_eq!(response.status(), StatusCode::NO_CONTENT);
     }
+
+    #[rstest]
+    async fn get_power_restrictions_list(
+        #[future] fast_rolling_stock: TestFixture<RollingStockModel>,
+    ) {
+        let app = create_test_service().await;
+        let rolling_stock = fast_rolling_stock.await;
+        let response = call_service(
+            &app,
+            TestRequest::get()
+                .uri("/rolling_stock/power_restrictions")
+                .to_request(),
+        )
+        .await;
+        let power_restrictions = rolling_stock
+            .model
+            .power_restrictions
+            .clone()
+            .unwrap()
+            .unwrap();
+        assert!(!power_restrictions.is_null());
+        let response_body: Vec<String> = read_body_json(response).await;
+        assert_eq!(response_body.len(), 1);
```

Comment:
How about something resembling the following ? It would make the test not fail when running locally

```suggestion
        assert_eq!(response_body..contains("C2".to_string()));
```